app/src/main/java/com/tip/lunchbox/viewmodel/HomeViewModel.java

```diff
@@ -28,7 +40,7 @@
 
     public void FetchRestaurantLiveData() {
 
-        Repository repository = new Repository();
+        //repository = new Repository();
```

Comment:
Sorry forgot to remove this will Rebase changes please review other files till then